### PR TITLE
Modifying the OrderResourcePlugin to display correct data

### PR DESCRIPTION
**Pull request by Floflinou**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?   | no
| Fixed tickets | #211 

 The exported grid didn't display every information. Please review it and get back to me when possible !

### CODE REVIEW

File: src/Exporter/Plugin/OrderResourcePlugin.php [hunk 1]
```diff
@@ -57,6 +59,20 @@ public function init(array $idsToExport): void
...
+            $this->addDataForResource($resource, 'Checkout_completed_at', $resource->getCheckoutCompletedAt());
+            $this->addDataForResource($resource, 'Payment_state', $resource->getPaymentState());
+            $this->addDataForResource($resource, 'Checkout_state', $resource->getCheckoutState());
+            $this->addDataForResource($resource, 'Shipping_state',
+            $resource->getShippingState());
+            $this->addDataForResource($resource, 'Token_value',
+            $resource->getTokenValue());
+            $this->addDataForResource($resource, 'Customer_ip',
+            $resource->getCustomerIp());
+            /** @var OrderInterfaceB $resourceB */
```
Comment (oallain) [inline src/Exporter/Plugin/OrderResourcePlugin.php]:
```suggestion
```

File: src/Exporter/Plugin/OrderResourcePlugin.php [hunk 2]
```diff
@@ -57,6 +59,20 @@ public function init(array $idsToExport): void
...
+            $this->addDataForResource($resource, 'Notes',
+            $resourceB->getNotes());
```
Comment (oallain) [inline src/Exporter/Plugin/OrderResourcePlugin.php]:
```suggestion
            $resource->getNotes());
```

File: src/Exporter/Plugin/OrderResourcePlugin.php [hunk 3]
```diff
@@ -8,6 +8,7 @@
 use FriendsOfSylius\SyliusImportExportPlugin\Exporter\ORM\Hydrator\HydratorInterface;
 use FriendsOfSylius\SyliusImportExportPlugin\Service\AddressConcatenationInterface;
 use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Order\Model\OrderInterface as OrderInterfaceB;
```
Comment (oallain) [inline src/Exporter/Plugin/OrderResourcePlugin.php]:
```suggestion
```

File: src/Exporter/Plugin/OrderResourcePlugin.php [hunk 4]
```diff
@@ -47,6 +48,7 @@ public function init(array $idsToExport): void
         foreach ($this->resources as $resource) {
             // insert customer information to specific fields
             $this->addCustomerData($resource);
+
```
Comment (oallain) [inline src/Exporter/Plugin/OrderResourcePlugin.php]:
```suggestion
```

**Comment by oallain** [comment]
Hi @Floflinou 

Can you accept my 2 suggested changes ? You can see it in Travis build like https://travis-ci.org/FriendsOfSylius/SyliusImportExportPlugin/jobs/639822026

Thanks

**Comment by Floflinou** [comment]
> Hi @Floflinou
> 
> Can you accept my 2 suggested changes ? You can see it in Travis build like https://travis-ci.org/FriendsOfSylius/SyliusImportExportPlugin/jobs/639822026
> 
> Thanks

Hi @oallain 

Of course, but how could I do that please ?

Thanks !

**Comment by oallain** [comment]
> > Hi @Floflinou
> > Can you accept my 2 suggested changes ? You can see it in Travis build like https://travis-ci.org/FriendsOfSylius/SyliusImportExportPlugin/jobs/639822026
> > Thanks
> 
> Hi @oallain
> 
> Of course, but how could I do that please ?
> 
> Thanks !

Hello,

You no longer need to do it.
I made the modification directly online with the Github editor (this is the first time I test this :)).

**Comment by oallain** [comment]
Thanks @Floflinou !

**Pull request merged**